clang/docs/ReleaseNotes.rst

```diff
@@ -280,6 +280,8 @@ Bug Fixes to C++ Support
   was only accepted at namespace scope but not at local function scope.
 - Clang no longer tries to call consteval constructors at runtime when they appear in a member initializer.
   (`#782154 <https://github.com/llvm/llvm-project/issues/82154>`_`)
```

Comment:
I think this should be:

```suggestion
  (`#82154 <https://github.com/llvm/llvm-project/issues/82154>`_`)
```

Which would be easy to fix as part of this PR, or a new one?

Reply:
Oups. New PR (Or nfc change, no need for a review for that)